server/lorax_server/utils/tokens.py

```diff
@@ -188,8 +192,9 @@ def from_pb(
         tokenizer: PreTrainedTokenizerBase,
     ) -> "StoppingCriteria":
         stop_sequence_criterias = [StopSequenceCriteria(sequence) for sequence in pb.stop_sequences]
+        eos_token_id = getattr(tokenizer, "_eos_token_ids", tokenizer.eos_token_id)
```

Comment:
this should be `getattr(tokenizer, "eos_token_ids", tokenizer.eos_token_id)` right?

Reply:
I was going back and forth but let's go with no _

Reply:
I see, because we're setting a custom property on the tokenizer? both make sense! Maybe _ might be better actually